src/js/components/productSupplier/create/subsections/FixedPrice.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import { Controller } from 'react-hook-form';
+
+import CheckBox from 'components/form-elements/v2/Checkbox';
+import DateField from 'components/form-elements/v2/DateField';
+import TextInput from 'components/form-elements/v2/TextInput';
+import Subsection from 'components/Layout/v2/Subsection';
+
+const FixedPrice = ({ control, errors }) => (
+  <Subsection
+    title={{
+      label: 'react.productSupplier.form.subsection.fixedPrice',
+      defaultMessage: 'Fixed Price',
+    }}
+    collapsable={false}
+  >
+    <div className="form-grid-3">
+      <Controller
+        name="contractPriceEach"
```

Comment:
let's name it as `contractPricePrice`, because this is how currently it is named on the backend logic for import product sources. 
there is an association in `ProductSupplier`:
`ProductPrice contractPrice`, so I think that `contractPriceEach` would not be intuitive in this case